packages/fluentui/docs/src/examples/components/Tree/Usage/TreMultiselectExample.shorthand.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import { Tree } from '@fluentui/react-northstar';
+
+const items = [
+  {
+    id: 'tree-item-1',
+    title: 'House Lannister',
+    items: [
+      {
+        id: 'tree-item-11',
+        title: 'Tywin',
+        selectionIndicator: { content: 'select all' },
+        styles: { selectionIndicatorMargin: '0 3rem 0 0' },
```

Comment:
is this correct? shouldn't this be `variables`?

Reply:
it is already in variable, just I need find proper usage :)

Reply:
Proper usage was 
  title: { content: 'Tywin', variables: { selectionIndicatorMargin: '0 10rem 0 0' } },
when I tested it it was working properly.

In the end @mnajdova recommended to me not use variable in usage example. I will follow her advice. 
Thank you to Marija for helping me clarify this part.